docs/docs/lib/csharp.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: C# SDK
+description: GrowthBook SDK for C#
+sidebar_label: C#
+slug: C#
+---
+
+# C#
+
+View the full documentation on [GitHub](https://github.com/growthbook/growthbook-c-sharp).
+
+[![nuget badge for growthbook](https://img.shields.io/nuget/v/growthbook-c-sharp?style=flat-square)](https://www.nuget.org/packages/growthbook-c-sharp)
+
+## Installation
+
+```csharp
```

Comment:
To make this highlighting work, we need to add `"csharp"` to the `additionalLanguages` array in `docusaurus.config.js` on line 136